app/admin/dashboard.rb

```diff
@@ -161,7 +160,145 @@
         script do
           raw "$(document).ready(function(){$('tr').attr('id', '')});"
         end
+      end # tab
+
+      tab :metrics do
+        panel 'General Traffic' do
+          site_visit_stats = Ahoy::Event.where(name: 'Site visit').where("properties->>'ip_address' is not null").where(time: @beginning_of_last_month..@end_of_last_month).group("properties->>'ip_address'").count
+          general_traffic_stats = [{
+                                    total_accounts: User.all.count,
+                                    unique_visitors: site_visit_stats.keys.length,
```

Comment:
❓ not sure the best way (or if it's possible to test this locally??) 🧐

Reply:
i tested this locally by making my local site public through a tunneling software and hitting my own site, as well as enlisting alex to help